website/docs/language/settings/backends/s3.mdx

```diff
@@ -410,12 +410,35 @@ to only a single state object within an S3 bucket is shown below:
 }
 ```
 
-It is not possible to apply such fine-grained access control to the DynamoDB
-table used for locking, so it is possible for any user with Terraform access
-to lock any workspace state, even if they do not have access to read or write
-that state. If a malicious user has such access they could block attempts to
-use Terraform against some or all of your workspaces as long as locking is
-enabled in the backend configuration.
+It is also possible to apply fine-grained access control to the DynamoDB
```

Comment:
I love this! My only nitpick is that I like to say something like "The following example policy [description of what it does]."  Could we add something like that? Otherwise, I fully support the addition of an example - thank you so much for doing this!

Reply:
Thanks for this, @noce2. I'll echo @laurapacilio's comment about adding some more detail. For example, the section describing S3 fine-grained access says

> Amazon S3 supports fine-grained access control on a per-object-path basis
using IAM policy. A full description of S3's access control mechanism is
beyond the scope of this guide, but an example IAM policy granting access
to only a single state object within an S3 bucket is shown below:

It would be a good idea to link to the AWS documentation on DynamoDB fine-grained locking at https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/specifying-conditions.html.

I would also explain why the IAM policy needs to access both `myorg-terraform-states/myapp/production/tfstate` and `myorg-terraform-states/myapp/production/tfstate-md5`.

Reply:
thanks for the comments all 🥳 , will make some updates